wscript

```diff
@@ -84,6 +86,54 @@ def _setup_openmp(conf):
     else:
         conf.end_msg("OpenMP found.", 'GREEN')
 
+def _ok_color(var, val):
+    if var == val:
+        return "GREEN"
+    else:
+        return "YELLOW"
+
+def _warn_color(var, val):
+    if var == val:
+        return "YELLOW"
+    else:
+        return "GREEN"
+
+def _err_color(var, val):
+    if var == val:
+        return "RED"
+    else:
+        return "GREEN"
+
+def _get_git_details(conf):
+    global APPNAME, VERSION, COMMIT, BRANCH
+
+    results = subprocess.run('git describe --dirty', shell=True, check=False, universal_newlines=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Sorry, I'm really not sure what you are on about since you can _only_ build from source and this PR only involves building from source.

What python packages (in HSTCAL?) or are you wondering if we can do the same thing for the wrappers or our python tools?